packages/ui/src/styles/globals.css

```
@@ -20,6 +31,61 @@
+   ======================== */
+
+/* 공통 높이 설정 */
+.map-container,
+.map-page,
+body.map-page-active {
+  height: 100vh !important;
+  height: 100dvh !important;
+  overflow: hidden !important;
+}
+
+/* map 페이지 전용 스타일 - 헤더/푸터 제외한 높이 */
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)